statsmodels/tsa/statespace/representation.py

```diff
@@ -237,19 +237,19 @@ class Representation(object):
     """
     transition = MatrixWrapper('transition', 'transition')
     r"""
-    (array) Transition matrix: :math:`T~(k\_states \times k\_states \times nobs)`
+    (array) Transition matrix: :math:`T~(k\_states \times k\_states \times nobs)`  # noqa E501
```

Comment:
I've found that E501 on docstrings is one of the few cases where it doesn't have to be on the actual too-long line (especially helpful for docstrings where many lines are too long).  Instead it can go after the closing `"""`.

I'm not sure if it matters for the flake parser, but for internal consistency I've been doing `# noqa:[EW]\d{3}`